diff --git a/commerce_kickstart.info b/commerce_kickstart.info
index a4143d0..15a0416 100644
--- a/commerce_kickstart.info
+++ b/commerce_kickstart.info
@@ -78,7 +78,6 @@ dependencies[] = entityreference
 dependencies[] = views_megarow
 dependencies[] = commerce_addressbook
 dependencies[] = commerce_discount
-dependencies[] = commerce_discount_date
 dependencies[] = commerce_add_to_cart_confirmation
 dependencies[] = commerce_message
 dependencies[] = commerce_backoffice
diff --git a/drupal-org.make b/drupal-org.make
index 1ff31d8..1d93f62 100644
--- a/drupal-org.make
+++ b/drupal-org.make
@@ -35,7 +35,10 @@ projects[commerce_autosku][download][revision] = 32e86f4
 projects[commerce_autosku][download][branch] = 7.x-1.x
 projects[commerce_migrate][version] = 1.2
 projects[commerce_migrate][patch][2701333] = https://www.drupal.org/files/issues/reference_fields_should-2701333-3.patch
-projects[commerce_discount][version] = 1.0-alpha7
+;projects[commerce_discount][version] = 1.0-alpha7
+projects[commerce_discount][download][type] = git
+projects[commerce_discount][download][branch] = 7.x-1.x
+projects[commerce_discount][download][revision] = 185aa94
 projects[commerce_checkout_progress][version] = 1.4
 projects[commerce_extra_price_formatters][version] = 1.x-dev
 projects[commerce_extra_price_formatters][download][type] = git
